client/blocks/conversation-follow-button/style.scss

```diff
@@ -0,0 +1,78 @@
+$conversation-follow-button-gray-disabled: lighten( $gray, 20% );
```

Comment:
I forget when we need this? Or do we?

Reply:
We do not! Good spot. That's left over from the site follow button.

Reply:
so if we don't need it can we remove it?

Reply:
I thought I had, but apparently didn't commit it! Gone now.